internal/horusec-javascript/ast.go

```diff
@@ -386,6 +387,21 @@ func (p *parser) parseStmt(node *cst.Node) ast.Stmt {
 		}
 
 		return stmt
+	case ForStatement:
+		return &ast.ForStatement{
+			Position:  ast.NewPosition(node),
+			VarDecl:   p.parseStmt(node.ChildByFieldName("initializer")),
+			Cond:      p.parseStmt(node.ChildByFieldName("condition")),
+			Increment: p.parseExpr(node.ChildByFieldName("increment")),
+			Body:      p.parseFuncBody(node.ChildByFieldName("body")),
+		}
+	case ForInStatement:
+		return &ast.ForInStatement{
+			Position: ast.NewPosition(node),
+			Left:     ast.NewIdent(node.ChildByFieldName("left")),
+			Right:    ast.NewIdent(node.ChildByFieldName("right")),
```

Comment:
Changing these fields to Expr, this should be changed to `p.parseExpr(node.ChildByFieldName("...")`